.github/workflows/tests.yml

```
@@ -128,12 +127,19 @@ jobs:
             ${{ runner.os }}-${{matrix.node-version}}-${{matrix.graphql_version}}-jest-
       - name: Build
         run: npm run build
-      - name: ${{matrix.name}} Tests
-        uses: nick-fields/retry@ad984534de44a9489a53aefd81eb77f87c70dc60 # v4
-        with:
-          timeout_minutes: 10
-          max_attempts: 5
-          command: npm run ${{matrix.name == 'Leak' && 'test:leaks' || 'test'}} --ci
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)